# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/xalan-c/no-tests.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- xalan_c-1.12/CMakeLists.txt.vanilla	2025-01-26 16:28:31.619999900 +0100
+++ xalan_c-1.12/CMakeLists.txt	2025-01-26 16:28:36.239999901 +0100
@@ -97,8 +97,6 @@
 add_subdirectory(src/xalanc)
 add_subdirectory(src/xalanc/TestXSLT)
 add_subdirectory(src/xalanc/TestXPath)
-add_subdirectory(samples)
-add_subdirectory(Tests)
 add_subdirectory(docs/doxygen)
 
 # Display configuration summary
